templates/single.twig

```diff
@@ -38,10 +38,11 @@
 					<div class="col-md-6">
 						<div class="single-post-meta">
 							<address class="single-post-author">
-								{{ __( 'by', 'planet4-master-theme' ) }}
```

Comment:
This change is made because of issue found on below link -
https://k8s.p4.greenpeace.org/test-venus/story/28598/why-you-should-care-where-your-seafood-comes-from/
![image](https://user-images.githubusercontent.com/5357471/79842985-3e9e7200-83d7-11ea-82c3-21a51cf5cd54.png)

Reply:
Shouldn't you wrap this in `{% if post.author %}`? On the test instance it looks like this post doesn't have an author set, so with the current change this will still be showing as it happens in both branches of the if-else below.

Reply:
You are right, I updated the if condition. 👍